src/PostgREST/Config.hs

```diff
@@ -233,6 +235,7 @@ parser :: Maybe FilePath -> Environment -> [(Text, Text)] -> RoleSettings -> Rol
 parser optPath env dbSettings roleSettings roleIsolationLvl =
   AppConfig
     <$> parseAppSettings "app.settings"
+    <*> (fromMaybe "*" <$> optString "cors-allowed-origins")
```

Comment:
Maybe it's better to use the `splitOnCommas` function here (see other examples for other configs below). The type of this AppConfig could be `Maybe [Text]`, if it's not set then it means `Nothing` for all origns (`*`).